internal/database/database.go

```diff
@@ -0,0 +1,606 @@
+	//queryauth.Scan(&result.authed)
+	return "Login success", nil
+}
+
+func (db *PGDataBase) CheckAuth(login string, ctx context.Context) (string, error) {
+	log.Println("Check Auth Start")
+	sqlGetStatus := `SELECT login,is_authed FROM users WHERE login=$1 FETCH FIRST ROW ONLY;`
+	result := GetUserData{}
+	query := db.conn.QueryRowContext(ctx, sqlGetStatus, login)
+	query.Scan(&result.login, &result.authed)
+	if result.login == "" {
+		return "", handlers.NewErrorWithDB(errors.New("not found"), "user not found")
+	}
+	if result.authed == false {
+		return "", handlers.NewErrorWithDB(errors.New("not authed"), "user not authed")
+	}
+	return "ok", nil
+}
+
+func (db *PGDataBase) Register(login string, pass string, ctx context.Context) error {
+	log.Println("Start Register")
+	sqlAddUser := `INSERT INTO users (login, password)
+				  VALUES ($1, $2)`
+
+	_, err := db.conn.ExecContext(ctx, sqlAddUser, login, pass)
+
+	if err, ok := err.(*pq.Error); ok {
+		log.Println("DB REGISTER ERROR HERE")
+		if err.Code == pgerrcode.UniqueViolation {
+			log.Println("UniqueViolation")
+			return handlers.NewErrorWithDB(err, "Conflict")
+		}
+		log.Println(err)
+	}
+	sqlAddWallet := `INSERT INTO wallet (owner, current_value, withdrawed)
+				  VALUES ($1, $2, $3)`
+
+	_, err = db.conn.ExecContext(ctx, sqlAddWallet, login, "0", "0")
+	if err != nil {
+		log.Println("err db.conn.180 Reg ", err)
+		return err
+	}
+	log.Println("err is nil")
+	return err
+}
+func (db *PGDataBase) UploadOrder(login string, order string, ctx context.Context) error {
+	sqlCheckOrder := `SELECT owner FROM orders WHERE order_temp=$1 FETCH FIRST ROW ONLY;`
+
+	result := GetUserData{}
+	query := db.conn.QueryRowContext(ctx, sqlCheckOrder, order)
+	err := query.Scan(&result.login) //or how check empty value?
+	if result.login != "" {
+		log.Println("DB ERROR OF UPLOAD ORDER")
+		log.Println(result.login)
+		log.Println(login)
+		if result.login == login {
+			log.Println("Alredy here")
+			return handlers.NewErrorWithDB(errors.New("Alredy here"), "Already here")
+		}
+		if result.login != login {
+			log.Println("Conflict")
+			return handlers.NewErrorWithDB(errors.New("Conflict"), "Conflict")
+		}
+
+	}
+
+	if err, ok := err.(*pq.Error); ok {
+		if err.Code == pgerrcode.NoData {
+			log.Println("pgerrcode.NoData")
+
+		}
+		if err.Code == pgerrcode.SuccessfulCompletion {
+			log.Println("pgerrcode.SuccessfulCompletion")
+		}
+		if err.Code == pgerrcode.CaseNotFound {
+			log.Println("pgerrcode.CaseNotFound")
+		}
+
+		log.Println(err)
+		return err
+	}
+
+	sqlAddOrder := `INSERT INTO orders (owner, order_temp,status)
+				  VALUES ($1, $2, $3)`
+
+	_, err = db.conn.ExecContext(ctx, sqlAddOrder, login, order, "NEW")
+
+	return err
+}
+
+type RespOrder struct {
+	Order      string
+	Status     string
+	Accrual    string
+	UploadedAt time.Time
+}
+
+func (db *PGDataBase) GetOrder(login string, ctx context.Context) ([]handlers.ResponseOrder, error) {
```

Comment:
plural form is better :)
```suggestion
func (db *PGDataBase) GetOrders(login string, ctx context.Context) ([]handlers.ResponseOrder, error) {
```